libs-scala/scala-utils/BUILD.bazel

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load(
+    "//bazel_tools:scala.bzl",
+    "da_scala_library",
+    "da_scala_test",
+    "lf_scalacopts",
+)
+
+hj_scalacopts = lf_scalacopts + [
```

Comment:
What does hj stand for?

Reply:
initially it was for Http Json I guess. I don't have a better name for it... suggestions?

If we use `"-P:wartremover:traverser:org.wartremover.warts.NonUnitStatements"` more widely, it makes sense to move it into `bazel_tools/scala.bzl` and place next to `lf_scalacopts` definition... but I am out of ideas what to call it :)

`lf_strict_scalaopts`???

Reply:
Given that you are only using it in this file, maybe just call it `scalacopts` without a prefix?

Reply:
:+1: